packages/calypso-build/webpack.config.js

```diff
@@ -107,7 +107,7 @@ function getWebpackConfig(
 			new webpack.IgnorePlugin( /^\.\/locale$/, /moment$/ ),
 			...SassConfig.plugins( { cssFilename, minify: ! isDevelopment } ),
 			new DuplicatePackageCheckerPlugin(),
-			new WordPressExternalDependenciesPlugin(),
+			...( env.WP ? [ new WordPressExternalDependenciesPlugin() ] : [] ),
```

Comment:
Because of this change, I suppose we need to add `WP` env in Jetpack before publishing the next version?

Reply:
Yep! Breaking change!